pkg/namesgenerator/names-generator.go

```diff
@@ -569,6 +578,9 @@ var (
 
 		// Ada Yonath - an Israeli crystallographer, the first woman from the Middle East to win a Nobel prize in the sciences. https://en.wikipedia.org/wiki/Ada_Yonath
 		"yonath",
+
+		// Konrad Zuse - a German engineer, inventor of the first programmable computer. https://en.wikipedia.org/wiki/Konrad_Zuse
```

Comment:
@thaJeztah I could replace zuse with another person and make an extra pr for zuse. This way it wouldnt delay merging the other names